docs/styleguide/PyGuide.md

```diff
@@ -801,7 +801,7 @@ def log_current_time(log_stream: LogStream):
     ...
 ```
 
-*checkpointing/converter.py*
+*checkpoint_converter.py*
```

Comment:
Looks like some paths could be affected

Reply:
no, it's just example in styleguide about naming